akka-docs/src/main/paradox/coexisting.md

```diff
@@ -0,0 +1,93 @@
+# Coexistence
+
+We believe Akka Typed will be adopted in existing systems gradually and therefore it's important to be able to use typed 
+and untyped actors together, within the same `ActorSystem`. Also, we will not be able to integrate with all existing modules in one big bang release and that is another reason for why these two ways of writing actors must be able to coexist.
+
+There are two different `ActorSystem`s: `akka.actor.ActorSystem` and `akka.actor.typed.ActorSystem`. 
+The latter should only be used for greenfield projects that are only using typed actors. The `akka.actor.ActorSystem`
+can be adapted so that it can create typed actors.
+
+Typed and untyped can interact the following ways:
+
+* untyped actor systems can create typed actors
+* typed actors can send messages to untyped actors, and opposite
+* spawn and supervise typed child from untyped parent, and opposite
+* watch typed from untyped, and opposite
+
+In the Scala examples the `akka.actor` package is aliased to `untyped`. The Java examples use fully quaified
```

Comment:
use the `@java[java things]` respective `@scala[scala truths]` rather than talking about both in the same text?

Reply:
:+1: